chapters/dashboard/index.md

```diff
@@ -1,87 +1,24 @@
-# Using Dashboard to Monitor Logs and View Flow
+# Jina Dashboard
 
-**Dashboard** (beta) offers the insight of running tasks in Jina. With dashboard, one can analyze logs, design flows and view Jina Hub images.
+**Jina Dashboard** is a visual interface to facilitate Development with Jina.
```

Comment:
You might consider replacing this with the new description Lucia came up with for our website. Not a requirement, just to keep [brandalignment](https://docs.google.com/document/d/1DVSUMRye5icmtwGCRBpwmvKgKtZOiczdVSH0NRg6xvQ/edit#)